app/build.gradle.kts

```diff
@@ -20,6 +22,9 @@ dependencies {
     implementation(projects.base)
     implementation(projects.feature.more)
 
+    implementation(platform(libs.firebase.bom))
+    implementation("com.google.firebase:firebase-crashlytics-ktx")
```

Comment:
please make in libs.toml

`firebase-crashlytics = { module = "com.google.firebase:firebase-crashlytics" }`